### PR TITLE
More unit tests for series

### PR DESCRIPTION
**Pull request by jhaugh42**

### CODE REVIEW

File: test/unit/series.js [hunk 1]
```diff
@@ -5,37 +5,189 @@ var expect = require('chai').expect;
...
+        var seriesCallbackBindResult = {whatever: 'was the bind result'};
+
+        before(function() {
+            seriesCallbackStub = sinon.stub(Series, '_seriesCallback');
+            seriesCallbackStub.bind = bindStub = sinon.stub();
+        });
+
+        beforeEach(function() {
+            seriesCallbackStub.reset();
+            seriesCallbackStub.bind = bindStub = sinon.stub();
```
Comment (enigmatic00) [inline test/unit/series.js:29]: if the stub is being reset on the line above, why re-assign it to a new stub?

File: test/unit/series.js [hunk 2]
```diff
@@ -5,37 +5,189 @@ var expect = require('chai').expect;
...
+            Series.series(inputFunctions, sinon.spy());
+            expect(function1.callCount).to.equal(1);
+        });
+
+        it('should not invoke the second function passed to the series', function() {
+            Series.series(inputFunctions, sinon.spy());
+            expect(function2.callCount).to.equal(0);
+        });
+
+        it('should bind the provided callback, funciton array and 1 to Series._seriesCallback', function() {
```
Comment (enigmatic00) [inline test/unit/series.js:48]: `function`

Comment (jhaugh42) [inline test/unit/series.js:48]: fixed in #3

File: test/unit/series.js (hunk 1 above)
Comment (jhaugh42) [inline test/unit/series.js:29]: addressed in #3

**Pull request merged**